src/build.props

```diff
@@ -10,8 +10,8 @@
     <Company Condition=" '$(Company)' == '' ">Microsoft</Company>
     <Product Condition=" '$(Product)' == '' ">Microsoft SARIF SDK</Product>
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
-    <VersionPrefix>4.2.0</VersionPrefix>
-    <PreviousVersionPrefix>4.1.0</PreviousVersionPrefix>
+    <VersionPrefix>4.2.1</VersionPrefix>
```

Comment:
>1 [](http://example.com/codeflow?start=23&length=1)

If we only have a behavioral bug fix, then this versioning change is good. Will keep my eyes open for a more serious change that might argue for upgrade to 4.3